roles/ceph-client/tasks/pre_requisite.yml

```diff
@@ -3,8 +3,8 @@
   copy:
     src: "{{ fetch_directory }}/{{ fsid }}/etc/ceph/{{ cluster }}.client.admin.keyring"
     dest: "/etc/ceph/"
-    owner: "ceph"
-    group: "ceph"
+    owner: "root"
```

Comment:
i think there is something wrong with that.
either the uid/gid or the mode should be changed?